docs/src/rules/consistent-this.md

```diff
@@ -69,7 +69,7 @@ foo.bar = this;
 
 Examples of **incorrect** code for this rule with the default `"that"` option, if the variable is not initialized:
 
-::: incorrect
+::: incorrect { "sourceType": "script" }
```

Comment:
This was to make the example "incorrect", but the bug reported in https://github.com/eslint/eslint/issues/19244 still exists.